package.json

```diff
@@ -14,7 +14,7 @@
     "babel-code-frame": "7.0.0-alpha.12",
     "babylon": "7.0.0-beta.22",
     "chalk": "2.1.0",
-    "cosmiconfig": "davidtheclark/cosmiconfig",
+    "cosmiconfig": "git://github.com/davidtheclark/cosmiconfig.git#6eb3cc85d3a435e6b9e252b341e601960312f410",
```

Comment:
I usually write these things like `github:davidtheclark/cosmiconfig#6eb3cc85d3a435e6b9e252b341e601960312f410`

Reply:
I wrote this way just for the consistency (L36, L39). 😅